### ISSUE TITLE
Focus zoom on selected object

### ISSUE THREAD

**Issue by noossyy** [bambulab/BambuStudio#1416]
**Is your feature request related to a problem? Please describe.**

Without a focus zoom on an object, your can’t properly rotate around and object after having zoomed on it. 

**Describe the solution you'd like**
Replicate the same behavior as the Z hotkey in superslicer or F hotkey in Autodesk Maya when object is selected. The camera will zoom close to it and center focus on it so you can properly rotate around it. 

**Describe alternatives you've considered**
Having a hotkey to do that like in most 3D animation software 

**Additional context**

**Comment by BowieInc** [comment]
Yes! How can I add my support for this shortcut addition? It is much needed.

**Comment by kertgartner** [comment]
YES PLEASE! Would love to see this added!

**Comment by Thelvaen** [comment]
I'd love to see this feature as well.

**Comment by CaidenKehrer** [comment]
I would also really appreciate this feature. It is nearly impossible to get angles right to view some objects .

**Comment by travrob1** [comment]
Yes please!!! That woulds same me so much time.  Thanks

**Comment by ctkjedi** [comment]
Another vote for this! Zoom to mouse is fine, but the moment you rotate your view...

**Comment by mpaperno** [comment]
Hi all,

Fun fact: Holding `CTRL` (`CMD` on Mac) while dragging with the left mouse button **will rotate around the current <strike>cursor position</strike> view's center point**!  Vs. the center of the current plate.

I just discovered this by looking at the code.  If it's documented somewhere, I can't find it!

It's not quite ideal since the rotation is still dependent on left mouse button, which means you can't click on the actual object you want to rotate around. Which is a general annoyance anyway.  But at least it's something.  Rotating round the center of the plate drives me nuts.

I'd now like the option of having this kind of rotation by default, with the CTRL modifier doing the opposite (rotating around plate center).

-Max
